CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 
 * Fixed `auto` strategy detected scanned document as having extractable text and using `fast` strategy, resulting in no output.
 * Fix list detection in MS Word documents.
+* Don't instantiate an element with a coordinate system when there isn't a way to get its location data.
```

Comment:
Could you also change the version in line one to 0.8.1-dev2, and change the version in `__version__.py` to match?